### PR TITLE
Updated ppp-extension.json

### PR DESCRIPTION
**Pull request by timgdavies**
The apply-extensions.py script we were using to build:

* The full PPP schema; and
* The merged ppp extension

uses json-merge-patch, which treats ```"field":null``` as removing that field.

Therefore, all the elements in the ppp release-schema that we wanted to include into the merged ppp extension which would null out an element, were getting lost. I've put in place a hack that does some find and replace on the incoming JSON files to replace ```null``` with ```"REPLACE_WITH_NULL"``` before merging for the ppp_extension (not for the full schema), and which then switches around the replacement before writing out the merged extension file.

### CODE REVIEW

**Comment by duncandewhurst** [comment]
@edugomez could we look at how we can resolve the conflicts here when we are in the office together on Friday?

I think the only change we actually need is the one to apply-extensions.py but that has been updated since Tim made this PR

**Comment by jpmckinney** [comment]
Marking this as a priority issue to review in next week's check-in.

File: docs/extensions/metrics.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ Metrics are structured like an [OLAP data cube](https://en.wikipedia.org/wiki/OL
 
 Where the metrics extension is used to model targets for a contracting process, the ```description``` field can be used to start whether the target is a minimum or recommended target.
 
+Where the metrics extension is used to model targets for a contracting process, the ```description``` field can be used to start whether the target is a minimum or recommended target.
```
Comment (jpmckinney) [inline docs/extensions/metrics.md]: Hmm, this is same as above paragraph.

**Comment by jpmckinney** [comment]
Closing in favor of #121.

**Pull request closed without merging**